src/DataGrid.php

```diff
@@ -3231,6 +3236,16 @@ public function setCustomPaginatorTemplate(string $templateFile): void
 	}
 
 
+	public function setGroupActionsConfirmDialog(string $confirmDialog): void
```

Comment:
I would rather see that cuntionality directly on `GroupActionCollection` rather that on the DataGrid class. Could you move it there? (see `DataGrid::getGroupActionCollection()`)

Reply:
@paveljanda  You are right, the setter is in the wrong place. I moved it now.